src/watchmaker/workers/salt.py

```diff
@@ -847,6 +850,15 @@ def install(self):
         if self.working_dir:
             self.cleanup()
 
+    def _policy_rpm(self, arg):
+        """Return name of distro version-appropriate policycoreutils RPM."""
+        if arg < '8':
+            result = 'policycoreutils-python'
+        elif arg >= '8':
```

Comment:
Is this second `elif` needed?  Can you just use `else`.  Seems redundant to check `>=` condition.

Reply:
I'd originally had the `elif` as an `arg = '8'` as a hedge against encountering deltas in 9.

…But then I remembered that my VPS is running CentOS 9-Stream and I checked to see if `9`'s RPM was different than `8`'s was. It's not different, so I changed the `=` to `>=` as a nod to the possibility that an eventual EL10 could be (given that, by the time EL10 comes out, Python3 is likely long-since retired).

I'm fine changing it or leaving it as a "logical-stub" type of declaration.

Reply:
Ok.  We can just leave as is.